lib/internal_metric/global.rb

```diff
@@ -339,5 +344,30 @@ def find_stuck_sidekiq_jobs
       end
       stats
     end
+
+    PG_HIGHEST_SEQ_CHECK_SECONDS = 60
+
+    def calc_pg_highest_seq
+      @@pg_highest_seq_last_check ||= 0
+
+      if @@pg_highest_seq_last_check >= Time.now.to_i - PG_HIGHEST_SEQ_CHECK_SECONDS
+        return @@pg_highest_seq_cache
+      end
+
+      @@pg_highest_seq_last_check = Time.now
+
+      value = (DB.query_single <<~SQL)[0]
```

Comment:
Do we need to loop through all the multisite databases here? Otherwise, we're just going to be collecting metrics on the main site.

Reply:
Right, very good point. Fixed